tweakwcs/multivariate.py

```diff
@@ -0,0 +1,234 @@
+"""
+Initial implementation of a function to fit multiple outputs 
+using astropy.modeling
+
+Notes
+-----
+
+This presumes that the multiple outputs are given equal weight in
+computing the least squares value. A future version may allow 
+specifying different weights when the quantities involved are
+not directly comparable. The motivating case for developing this
+was to compute the proper transformation of 2D coordinates, and as
+such, the least square computation is equivalent to computing the
+least squares radial differences in position. This could be generalized
```

Comment:
**radial differences** sounds confusing: is it a list of differences like `x1 - x1_0`, `x2 - x2_0`, ...,  `y1 - y1_0`, `y2 - y2_0`, ... **OR** is it like `r1 - r1_0`, `r2 - r2_0`, ... ?